sprint-planning/readme.md

```diff
@@ -1,29 +1,83 @@
 # Planning
 
-## Content TODO's
+## Goals
 
-- Complete outline below
-- Determining team capacity
-- Tasking sprint stories
-  - size of tasks
-- WIP limits / dividing up into workstreams
+During the [sprint planning](https://www.agilealliance.org/glossary/sprint-planning), the team discusses and agrees on the scope for the upcoming sprint.
 
-## Goals
+**Goals are:**
+
+- Selecting the **stories** that will be implemented in the sprint.
+- Estimating the **effort** required for the stories in the sprint.
+- Splitting the stories into **tasks**.
+
+General guidance:
 
-What are the intended outcomes of the ceremonies?
+- The sprint planning should happen at the begining of the new sprint (or at the end of the previous one). It usually last bewteen 1 and 4 hours depending on the size of the team and duration of the sprints.

```

Comment:
I would question planning lasting longer than 2 hours....would be interested in hearing what others think.

Reply:
I think the "depending on the size of the team and duration of the sprints"...if you have 4 week sprints (not CSE) you might need four hours of planning...one week sprints not so much.

Reply:
I am happy to reduce this. 4 hours at needed in 4 weeks sprints. If this only target what we are doing in CSE today (1 week sprints), then it makes more sense to on indicate 1-2 hours